libs/core/build.gradle

```diff
@@ -13,19 +13,19 @@ apply plugin: 'elasticsearch.mrjar'
 dependencies {
   // This dependency is used only by :libs:core for null-checking interop with other tools
   compileOnly "com.google.code.findbugs:jsr305:3.0.2"
-  compileOnly project(':libs:elasticsearch-logging')
+  compileOnly project(':libs:logging')
 
   testImplementation "com.carrotsearch.randomizedtesting:randomizedtesting-runner:${versions.randomizedrunner}"
   testImplementation "junit:junit:${versions.junit}"
   testImplementation "org.hamcrest:hamcrest:${versions.hamcrest}"
 
   testImplementation(project(":test:framework")) {
-    exclude group: 'org.elasticsearch', module: 'elasticsearch-core'
+    exclude group: 'org.elasticsearch', module: 'core'
```

Comment:
Does the gradle "module name" matter? This PR produces the same artifact ids, though the "name" attribute in the pom is different. Is there a way to control this module name in gradle? I could only find archivesName which we set as the artifactId.